src/services/PeopleSearchService.ts

```diff
@@ -119,18 +119,17 @@ export default class SPPeopleSearchService {
           PrincipalSource: 15,
           PrincipalType: this.getSumOfPrincipalTypes(principalTypes),
           QueryString: query,
-          SharePointGroupID: null
         }
       };
 
       // Search on the local site when "0"
       if (siteUrl) {
-        searchBody.queryParams.SharePointGroupID = 0;
+        searchBody.queryParams["SharePointGroupID"] = 0;
       }
 
       // Check if users need to be searched in a specific SharePoint Group
       if (groupId && typeof (groupId) === 'number') {
-        searchBody.queryParams.SharePointGroupID = groupId;
+        searchBody.queryParams["SharePointGroupID"] = groupId;
       }
```

Comment:
Can you revert to using dot notation, please? I assume this was not related to the issue itself?

Reply:
@joelfmrodrigues  in the previous version (3.9.0) it was using the [] syntax. Since SharePointGroupID is not defined on the queryParams object on line 114, the dot syntax throw an error on 127 and 132. The issue here is that you cannot pass a null value for SharePointGroupID, so we cannot include it on the queryParams object by default, which is why I believe the [] syntax is being used on 127 and 132. I'm happy to update the PR, just let me know what you think about the above.  Thanks!

Reply:
Sorry, you are correct as this would throw an error :) : https://stackoverflow.com/questions/12710905/how-do-i-dynamically-assign-properties-to-an-object-in-typescript

Any chance you could add `any` as the type for searchBody as this should allow the dot notation to be used? If it's too much hassle, let me know and I can update after approving the PR
Many thanks a sorry for so many comments

Reply:
@joelfmrodrigues not a problem, just made the changes and added a new commit. Let me know if anything else is needed. Thanks!